### PR TITLE
Renamed Unit Cell Matrices

### LINKED ISSUE #73: Renaming unit cell matrices
Rename unit cell matrices in `LBSSolver` to more informative names.

Refs #66

### PR DESCRIPTION
**Pull request by zhardy-lanl**
A quick PR to make the unit cell matrices have intuitive names.

Closes #73

### CODE REVIEW

File: modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc [hunk 1]
```diff
@@ -594,8 +594,8 @@ DiffusionMIPSolver::AssembleAand_b(const std::vector<double>& q_vector)
     const auto cc_nodes = cell_mapping.GetNodeLocations();
     const auto& unit_cell_matrices = unit_cell_matrices_[cell.local_id_];
 
-    const auto& cell_K_matrix = unit_cell_matrices.K_matrix;
-    const auto& cell_M_matrix = unit_cell_matrices.M_matrix;
+    const auto& cell_K_matrix = unit_cell_matrices.intV_gradshapeI_gradshapeJ;
+    const auto& cell_M_matrix = unit_cell_matrices.intV_shapeI_shapeJ;
```
Comment (andrsd) [inline modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc:598]: If you changed the name in `unit_cell_matrices`, should you also rename the variable, i.e. the `const auto &` one?

**Review by ragusa — APPROVED** [review]

File: modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc (hunk 1 above)
Comment (zhardy-lanl) [reply modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc]: Good catch. I used did a bulk rename in CLion but did not check for local variables. This has now been changed.

**Review by ragusa — APPROVED** [review]
3 small requests.

**Review by andrsd — APPROVED** [review]

**Comment by ragusa** [comment]
@zhardy-lanl I can't see my 3 little comments anymore. I guess I forgot to click add review or something. Did you see them? It was about moving qg[j] in 3 products to the the second term of the multiplication instead of being the first, as I tend to write a_ij x_j more seamlessly than x_j a_ij .

**Pull request merged**